File: app/src/main/java/com/karthik/myweather/utils/UIUtils.java
package com.karthik.myweather.utils;

public class UIUtils {

    private static final String url = "https://www.metaweather.com/static/img/weather/png/%s.png";

    public static String getImgUrl(String weatherCondition){
        return String.format(url, weatherCondition);
    }
}

File: app/src/main/java/com/karthik/myweather/utils/UIUtils.kt
package com.karthik.myweather.utils

object UIUtils {
    private const val url = "https://www.metaweather.com/static/img/weather/png/%s.png"
    @JvmStatic
    fun getImgUrl(weatherCondition: String?): String {
        return String.format(url, weatherCondition)
    }
}